package.json

```diff
@@ -63,6 +63,7 @@
     "@types/vuelidate": "^0.7.15",
     "@typescript-eslint/eslint-plugin": "^5.38.0",
     "@typescript-eslint/parser": "^5.38.0",
+    "@volar-plugins/vetur": "latest",
```

Comment:
What do you think about declaring a specific version number instead of "latest"?

Reply:
This is how Travis did it in [Auth Web](https://github.com/bcgov/sbc-auth/blob/main/auth-web/package.json) and how I did it in [Filings UI](https://github.com/bcgov/business-filings-ui/blob/main/package.json).

I think "latest" probably makes good sense since we should generally be using the latest VS Code extension versions anyway (and this dependency should match that).